ajax/libs/jquery.quicksearch/package.json

```diff
@@ -3,15 +3,15 @@
   "filename": "jquery.quicksearch.min.js",
   "description": "A jQuery plugin for searching through tables, lists, etc quickly",
   "version": "2.4.0",
-  "homepage": "https://deuxhuithuit.github.io/quicksearch/",
+  "homepage": "http://deuxhuithuit.github.io/quicksearch/",
```

Comment:
Https is good to access here.
It's safe for browsing.

Please roll back this line.
Thank you.